x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/integrations/const.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { CategoryFacet } from '@kbn/fleet-plugin/public';
+import type { Tab } from './types';
+
+export const ADD_AGENT_PATH = `/agents`;
+export const AGENT_INDEX = `logs-elastic_agent*`;
+export const AGENTLESS_LEARN_MORE_LINK = null; // Link to be confirmed.
+export const CARD_DESCRIPTION_LINE_CLAMP = 3; // 3 lines of text
+export const CARD_TITLE_LINE_CLAMP = 1; // 1 line of text
+export const FLEET_APP_ID = `fleet`;
+export const INTEGRATION_APP_ID = `integrations`;
+export const LOADING_SKELETON_HEIGHT = 10; // 10 lines of text
+export const MAX_CARD_HEIGHT = 127; // px
+export const ONBOARDING_APP_ID = 'onboardingAppId';
+export const ONBOARDING_LINK = 'onboardingLink';
+export const SCROLL_ELEMENT_ID = 'integrations-scroll-container';
+export const SEARCH_FILTER_CATEGORIES: CategoryFacet[] = [];
+export const WITH_SEARCH_BOX_HEIGHT = '517px';
+export const WITHOUT_SEARCH_BOX_HEIGHT = '462px';
+export const INTEGRATION_TABS: Tab[] = [
```

Comment:
I like the idea of the `INTEGRATION_TABS` config, it could be in its own file if you want